package.json

```diff
@@ -12,24 +12,19 @@
     "morgan": "^1.9.0"
   },
   "scripts": {
-    "build": "npm run clean && npm-run-all --parallel build:static build:node",
-    "build:node": "./tasks/build-node-scripts.sh",
-    "build:prod": "NODE_ENV=production npm run build",
-    "build:static": "./tasks/build-static.sh",
     "clean": "rm -rf dist",
-    "lint": "npm-run-all --parallel lint:eslint lint:stylelint",
-    "lint:eslint": "eslint 'src/**/*.js'",
+    "lint:eslint": "eslint src",
     "lint:stylelint": "stylelint 'src/**/*.css'",
-    "serve": "npm run build && npm-run-all --parallel watch server",
-    "start": "node dist/server/server.js",
-    "server": "node ./node_modules/.bin/nodemon node dist/server/server.js",
-    "test": "npm run test:node && npm run test:static",
-    "test:node": "./tasks/test-node.sh",
-    "test:static": "./node_modules/.bin/karma start",
-    "tools:import_products": "./node_modules/.bin/babel-node tools/import_products.js",
-    "watch": "npm-run-all --parallel watch:static watch:node",
-    "watch:node": "./node_modules/.bin/watch 'npm run build:node' src/server src/data",
-    "watch:static": "./node_modules/.bin/watch 'npm run build:static' src/client"
+    "lint": "npm-run-all --continue-on-error lint:eslint lint:stylelint",
+    "build:node": "./tasks/build-node-scripts.sh",
+    "build:static": "./tasks/build-static.sh",
+    "build": "npm-run-all clean build:node build:static",
+    "build:prod": "NODE_ENV=production npm run build",
+    "watch:dist": "nodemon --delay 2000ms --watch dist -e css,js,json,hbs,png,svg",
+    "watch:node": "nodemon --watch src/data --watch src/server --watch src/services --watch src/templates -e js,json,hbs --exec 'npm run build:node'",
+    "watch:static": "nodemon --watch src/client -e css,js,json,png,svg --exec 'npm run build:static'",
+    "start:watch": "npm-run-all --parallel watch:static watch:node watch:dist",
+    "start": "node dist/server/server.js"
```

Comment:
Can you address here the comments from #98 ? They still apply.

Reply:
Updated now to try and address this.